code/src/Components/Card.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Artists from "./Artists";
+import Release from "./Release";
+import AlbumImage from "./AlbumImage";
+import IconContainer from "./IconContainer";
+
+const Card = (props) => {
+  return (
+    <div className="album-card">
+      <div className="image-container">
+        <AlbumImage image={props.image} />
+        <IconContainer/>
```

Comment:
Your icons do not show when you hover over the album's photo.

Reply:
The hover effect is visible in the browser when I run npm start... but it's not in the deploy. I have tried to make another deploy but it's still not working... Any ideas what we should do?

Reply:
I have saved all files and commited and pushed several times.

Reply:
Posted a question here https://stackoverflow.com/c/technigo/questions/2306

Reply:
FIXED IT! With help from Katharina Liebig.